intro-HPC/examples/OpenFOAM/OpenFOAM_damBreak.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#PBS -l walltime=1:0:0
+#PBS -l nodes=1:ppn=4
+# check for more recent OpenFOAM modules with 'module avail OpenFOAM'
+module load OpenFOAM/5.0-20180108-foss-2018a
```

Comment:
@redfast00 Please use the latest `OpenFOAM` module that uses the `intel` toolchain here (and retest, check `interFoam.log`, you should clearly see whether it ran successfully)

Reply:
I changed it, now using intel-6 and my job is running. Will update once it completes.

Reply:
@boegel it worked, no warnings/errors anymore.